src/wallet/wallet2.h

```diff
@@ -673,16 +674,32 @@ namespace tools
     struct unsigned_tx_set
     {
       std::vector<tx_construction_data> txes;
-      std::pair<size_t, wallet2::transfer_container> transfers;
-      std::pair<size_t, std::vector<wallet2::exported_transfer_details>> new_transfers;
+      std::tuple<uint64_t, uint64_t, wallet2::transfer_container> transfers;
+      std::tuple<uint64_t, uint64_t, std::vector<wallet2::exported_transfer_details>> new_transfers;
 
       BEGIN_SERIALIZE_OBJECT()
-        VERSION_FIELD(1)
+        VERSION_FIELD(2)
         FIELD(txes)
-        if (version >= 1)
-          FIELD(new_transfers)
-        else
-          FIELD(transfers)
+        if (version == 0)
```

Comment:
Agreed for 0 and 1, new_transfers is the one exposed in the struct though, I think it's better left alone, since it's really 2+ and would need changing various other places in wallet2.